Stack.cs

```diff
@@ -0,0 +1,73 @@
+namespace ConsoleApp
+{
+    public class Stack<T>
+    {
+
+        public int count = 0;
```

Comment:
if it is public field - I can change it outside the stack, I don't think you want it  :)